### PR TITLE
Add keyboard navigation ability to Column Editor

### PR DESCRIPTION
**Pull request by sasumner**
Implements #8488 

Results (note underline characters, especially in _Text to Insert_ and _Number to Insert_):

image

### CODE REVIEW

File: PowerEditor/src/ScitillaComponent/columnEditor.rc [hunk 1]
```diff
@@ -36,27 +36,27 @@ EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
...
-    GROUPBOX        "Format",IDC_COL_FORMAT_GRP_STATIC,20,153,188,54,BS_CENTER
-    DEFPUSHBUTTON   "OK",IDOK,145,13,70,14,BS_NOTIFY
-    PUSHBUTTON      "Cancel",IDCANCEL,145,36,70,14,BS_NOTIFY
+    CONTROL         "&Dec",IDC_COL_DEC_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,30,169,70,10
+    CONTROL         "&Hex",IDC_COL_HEX_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,124,169,70,10
+    CONTROL         "&Oct",IDC_COL_OCT_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,30,188,70,10
+    CONTROL         "&Bin",IDC_COL_BIN_RADIO,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,124,188,70,10
+    GROUPBOX        "&Format",IDC_COL_FORMAT_GRP_STATIC,20,153,188,54,BS_CENTER
+    DEFPUSHBUTTON   "O&K",IDOK,145,13,70,14,BS_NOTIFY
+    PUSHBUTTON      "&Cancel",IDCANCEL,145,36,70,14,BS_NOTIFY
```
Comment (donho) [inline PowerEditor/src/ScitillaComponent/columnEditor.rc:61]: OK & Cancel are triggered by `Enter` and `ESC`, so it's not necessary to add the above 2 lines.

**Comment by sasumner** [comment]
@donho said:

>OK & Cancel are triggered by Enter and ESC

I will remove the accelerators for those two.

File: PowerEditor/installer/nativeLang/english.xml [hunk 2]
```diff
@@ -970,19 +970,19 @@ You can define several column markers by using white space to separate the diffe
...
-                <Item id="2031" name="Increase by :"/>
-                <Item id="2035" name="Leading zeros"/>
-                <Item id="2036" name="Repeat :"/>
-                <Item id="2032" name="Format"/>
-                <Item id="2024" name="Dec"/>
-                <Item id="2025" name="Oct"/>
-                <Item id="2026" name="Hex"/>
-                <Item id="2027" name="Bin"/>
-                <Item id="1" name="OK"/>
-                <Item id="2" name="Cancel"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/english.xml:985]: OK & Cancel entries should not be modified as well in english.xml

Comment (sasumner) [reply PowerEditor/installer/nativeLang/english.xml:985]: Agreed.
Sorry, I changed them in `nativeLang.xml` for testing but forgot `english.xml`.
:-(

**Pull request closed without merging**